models.py

```diff
@@ -41,3 +41,22 @@ class Skill:
     name: str
     proficiency: str
     logo: str
+
+@dataclass
+class Contact:
+    '''
+    Contact Class
+    '''
+    contact_id: int
+    name: str
+    phone: str
+    email: str
+
+    id_counter = 1
+
+    def __init__(self, name, phone, email):
+        self.contact_id = Contact.id_counter
+        Contact.id_counter += 1
```

Comment:
Every time the server is restarted the counter resets. It would be nice to keep the ID independent for future development. What do you think?

Reply:
I will fix it and remove the ID for now. The reason I used ID before was to make updating a contact easier to update, all you need to do is just pass the id and it will fetch the contact that has the id. I will look for a better way to do the update without id